### PR TITLE
Change filter icon to `HorizontalBarsFilterIcon` and change the search icon to `MagnifyingGlassIcon` in the advanced search

### PR DESCRIPTION
**Pull request by Achintha444**
### Purpose
$subject
Screenshot 2024-09-02 at 14 12 58

### Related Issues
- N/A

### Related PRs
- N/A

### Checklist
- [ ] e2e cypress tests locally verified. (for internal contributers)
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Relevant backend changes deployed and verified
- [ ] [Unit tests](/docs/testing/UNIT_TESTING.md) provided. (Add links if there are any)
- [ ] [Integration tests](https://github.com/wso2/product-is/tree/master/modules/integration) provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines
- [ ] Ran FindSecurityBugs plugin and verified report
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets

### CODE REVIEW

File: modules/react-components/src/components/input/advanced-search.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2020-2024, WSO2 LLC. (https://www.wso2.com).
```
Comment (dasuni-30) [inline modules/react-components/src/components/input/advanced-search.scss:2]:
```suggestion
 * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
```

File: .changeset/tricky-comics-fry.md
Comment (pavinduLakshan) [inline .changeset/tricky-comics-fry.md:1]: We need to specify all changed packages in the changeset.

Comment (Achintha444) [reply .changeset/tricky-comics-fry.md:1]: ack

Comment (Achintha444) [reply .changeset/tricky-comics-fry.md:1]: fixed with https://github.com/wso2/identity-apps/pull/6856/commits/7a883b06f98d74eef2f3f0edb2d4756bf838bdfc

**Review by pavinduLakshan — APPROVED** [review]

**Pull request merged**